EmaPajic/assignment1/assignment1_Q1.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+@author: EmaPajic
+"""
+
+import sys
+
+def areAnagrams(str1,str2,isCaseSensitive):
```

Comment:
Please use the right naming style for functions: lowercase words separated by underscores.
See https://www.python.org/dev/peps/pep-0008/#function-and-variable-names.

Reply:
updated :)